packages/browser/src/index.ts

```diff
@@ -375,7 +378,7 @@ export default class LogtoClient {
     accessToken,
     expiresIn,
   }: CodeTokenResponse) {
-    this.refreshToken = refreshToken;
+    this.refreshToken = refreshToken ?? null;
```

Comment:
Kinda forget. Do we have to use `null`?

Reply:
refreshToken is `Nullable<…>`. Have to convert `undefined` to `null`